app/assets/javascripts/components/dropdown-menu.js

```diff
@@ -1,14 +1,14 @@
 ManageIQ.angular.app.component('dropdownMenu', {
   bindings: {
-    id: '<',
+    widgetId: '<',
```

Comment:
One more bug.. this should be '@', otherwise you'll get a number (or undefined for compressed ids).

Reply:
Nooo.... I had it with `@` and changed it to original :D Fixed :)